### PR TITLE
minor spelling tweaks in md files

### PR DESCRIPTION
**Pull request by kiszk**
This PR addresses minor spelling tweaks in `.md` files. In addition, this PR addresses spelling tweaks in the following non-`.md` files.

```
tensorflow/core/api_def/base_api/api_def_QuantizeAndDequantizeV2.pbtxt
tensorflow/go/op/wrappers.go
```

### CODE REVIEW

File: tensorflow/lite/experimental/micro/examples/micro_speech/apollo3/README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@
         *   cmsis_power.txt: the magnitude squared of the DFT
         *   cmsis_power_avg.txt: the 6-bin average of the magnitude squared of
             the DFT
-    *   Run both verisons of the 1KHz pre-processor test and then compare.
+    *   Run both versons of the 1KHz pre-processor test and then compare.
```
Comment (jpienaar) [inline tensorflow/lite/experimental/micro/examples/micro_speech/apollo3/README.md]: versions ? (below too)

Comment (kiszk) [reply tensorflow/lite/experimental/micro/examples/micro_speech/apollo3/README.md]: Oh

**Review by jpienaar — APPROVED** [review]
Thanks!

**Review by liufengdb — APPROVED** [review]
lgtm!

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**